solidity/test/unit/Greeter.t.sol

```diff
@@ -74,7 +76,7 @@ contract UnitGreeterSetGreeting is Base {
   function test_EmitEvent(string memory _greeting) public {
     vm.assume(keccak256(bytes(_greeting)) != _emptyString);
 
-    _expectEmitNoIndex();
+    vm.expectEmit(false, false, false, true);
```

Comment:
Doesn't hurt to always check the indexed parameters and the source of the event.
```suggestion
    vm.expectEmit(true, true, true, true, address(_greeter));
```